src/Component/assets/data.js

```diff
@@ -0,0 +1,47 @@
+import p1_img from './Product1.png'
+import p2_img from './Product2.png'
+import p3_img from './Product3.png'
+import p4_img from './Product4.png'
+
+let data_product = [
```

Comment:
use const insted of let.
use camelCase for variable naming.
https://dev.to/sathishskdev/part-1-naming-conventions-the-foundation-of-clean-code-51ng